File: benchmarks/cross/progs/intloop.lua
local function compute(n)
  local acc = 0
  local i = 0
  while i < n do
    acc = acc + (i * 3 + 7) % 13
    acc = acc % 30011
    i = i + 1
  end
  return acc
end
print(compute(30000000))

File: benchmarks/cross/progs/intloop.js
function compute(n) {
  let acc = 0, i = 0;
  while (i < n) {
    acc = acc + (i * 3 + 7) % 13;
    acc = acc % 30011;
    i = i + 1;
  }
  return acc;
}
console.log(compute(30000000));
